docs/content/modeling/roles-and-permissions.mdx

```diff
@@ -21,16 +21,16 @@ import {
 
 <DocumentationNotice />
 
-In this guide you will learn how to model roles and permissions model within <ProductName format={ProductNameFormat.ProductLink}/> using the _<ProductConcept section="what-is-an-authorization-model" linkName="authorization model" />_ and _<ProductConcept section="what-is-a-relationship-tuple" linkName="relationship tuple" />_.
+This guide explains how to model roles and permissions in <ProductName format={ProductNameFormat.ProductLink}/> using <ProductConcept section="what-is-an-authorization-model" linkName="authorization models" /> and <ProductConcept section="what-is-a-relationship-tuple" linkName="relationship tuples" />. For the purpose of this guide:
 
-- **Roles** are assigned to <ProductConcept section="what-is-a-user" linkName="users" /> or a group of users, where any user can have more than one role (`editor`, `owner`, etc..).
-- **Permissions** are what allows users to access certain <ProductConcept section="what-is-an-object" linkName="objects" /> based on their specific roles (`device_renamer`, `channel_archiver`, etc..).
+- **Roles** are assigned to <ProductConcept section="what-is-a-user" linkName="users" /> or groups of users, where any user can have more than one role, like `editor` or `owner`.
+- **Permissions** allow users to access certain <ProductConcept section="what-is-an-object" linkName="objects" /> based on their specific roles, like `device_renamer` or `channel_archiver`.
 
-For example, the role `viewer` of a `trip` can have **permissions to view bookings** or the role `owners` can have **permissions to add/view bookings to a trip.**
+For example, the role `viewer` of a `trip` can have permissions to view bookings, while the role `owners` has permissions to add or view bookings to a trip.
 
 <CardBox title="When to use" appearance="filled">
 
-When trying to create a role and permissions model within <ProductName format={ProductNameFormat.ShortForm}/>.:
+When creating a role and permissions model within <ProductName format={ProductNameFormat.ShortForm}/>, you should:
 
 - Create roles by creating relations that can be directly assigned to users
```

Comment:
Use a period after a bullet list that completes the opening stem sentence that introduces it. Don't use a period after bullet lists that are not complete sentences or do not complete the opening stem sentence. Don't use semicolons to end punctuation. Use either all full sentences in your bullet lists or all fragments.